github/src/main/scala/org/typelevel/sbt/TypelevelScalaJSGitHubPlugin.scala

```diff
@@ -43,6 +44,9 @@ object TypelevelScalaJSGitHubPlugin extends AutoPlugin {
           s"$flag$l->$g"
         }
       }
+    },
+    scalaJSLinkerConfig := {
+      scalaJSLinkerConfig.value.withBatchMode(sys.env.get("GITHUB_ACTIONS").contains("true"))
```

Comment:
This plugin annoyingly doesn't know about the GH actions plugin. So it has to check for itself whether we are in CI.